### ISSUE TITLE
Split Maintenance commands

### ISSUE THREAD

**Issue by SebSept** [friends-of-presta/fop_console#37]
https://github.com/friends-of-presta/fop_console/blob/8a8f69f9e6b731abe16609961677e70ba42029ee/src/Commands/Maintenance.php#L29

This command handles 2 kinds of actions : change shop state & manage allowed ips for maintenance.
Maybe we could make 2 separates commands.

Don't you think ?

**Comment by jf-viguier** [comment]
Yes, I agree. Should be fop:console:maintenance-ip or something.

**Comment by tom-combet** [comment]
It doesn't really shock me since all the commands concern maintenance mode but I would be okay to split though.
So it would be `fop:shop:maintenance-status` and `fop:shop:maintenance-ip`?

**Comment by SebSept** [comment]
actions are : 
- get shop maintenance status
- set shop maintenance status
- get shop maintenance ip
- set shop maintenance ip

**Comment by tom-combet** [comment]
Yep so what's the conclusion? 🤔
